package.json

```diff
@@ -73,6 +73,7 @@
     "source-map-explorer": "2.0.1",
     "start-server-and-test": "^1.10.0",
     "tsconfig-paths-webpack-plugin": "^3.2.0",
+    "ts-loader": "^6.2.1",
```

Comment:
so we have `awesome-typescript-loader` and `ts-loader` -> not a big deal, but im curious why we need both

Reply:
Looks like we're using `awesome-typescript-loader` over in our packages, and `ts-loader` is the one that got added in our example app.. Interesting... I think it's definitely worth doing some compare/contrast and deciding on just one to use for both. It would feel weird if TS worked slightly differently in our tests vs our components.

`ts-loader` is just what I saw from other examples. I'm curious if there was a reason we went with `awesome-typescript-loader` for our components?

Reply:
i think it was just faster for storybook